app/institution/institution.js

```diff
@@ -13,6 +13,46 @@ Institution.prototype.make = function make() {
     };
     return institution;
 };
+
+Institution.prototype.isValid = function isValid() {
```

Comment:
no prototype de institution cria um json com os campos obrigatórios e faz um for dentro desse isValid, assim vc evita fazer tantos if's, até pq pode ser que tenha mais campos no futuro.